core/src/test/java/org/mskcc/oncotree/utils/CacheUtilTest.java

```diff
@@ -26,8 +26,8 @@
 import org.mskcc.oncotree.error.InvalidVersionException;
 import org.mskcc.oncotree.model.TumorType;
 import org.mskcc.oncotree.model.Version;
-import org.mskcc.oncotree.topbraid.TopBraidException;
-import org.mskcc.oncotree.topbraid.OncoTreeVersionRepository;
+import org.mskcc.oncotree.graphite.GraphiteException;
```

Comment:
If we can change these import statements but nothing else in the rest of the code of this class, doesn't it mean that the import statements are not needed because the class names are not referenced? (at least for GraphiteException)

Reply:
exception import dropped